draft-ietf-quic-transport.md

```diff
@@ -1530,12 +1532,31 @@ Stream ID:
 : ID of the stream whose flow control windows is being updated, or 0 to specify
   the connection-level flow control window.
 
-Byte offset:
+Flow Control Offset:
+
+: A 64-bit unsigned integer indicating the flow control offset for the given
+  stream (for a stream ID other than 0) or the entire connection.
+
+The flow control offset is expressed in units of octets for individual streams
```

Comment:
I considered it, but streams don't need as much space and they might benefit from a more granular expression of where they are up to.  We can refine this later.